docs/reference/logging/overview.md

```diff
@@ -172,3 +173,13 @@ The Amazon CloudWatch Logs logging driver supports the following options:
 
 
 For detailed information on working with this logging driver, see [the awslogs logging driver](awslogs.md) reference documentation.
+
+## Specify Splunk options
```

Comment:
Can you drop "Specify" here? It's more consistent with the other drivers (Amazon seems the odd one out here :smile:)